impeller/entity/inline_pass_context.h

```diff
@@ -42,18 +33,14 @@ class InlinePassContext {
 
   uint32_t GetPassCount() const;
 
-  RenderPassResult GetRenderPass(uint32_t pass_depth);
```

Comment:
This is just straight up dead code regardless of your change?  No need to change this PR but for future PRs it would be easier if refactoring and functional changes are split into 2 different prs =)

Reply:
Yeah I get it. Though I didn't intend to do a large refactoring, I realize all this was dead when I started changing it.